optimizely/decision/composite_service_test.go

```diff
@@ -76,10 +76,9 @@ func TestOnDecision(t *testing.T) {
 	testFeatureDecisionService := new(MockFeatureDecisionService)
 	testFeatureDecisionService.On("GetDecision", decisionContext, userContext).Return(expectedFeatureDecision, nil)
 
-	notificationCenter := notification.NewNotificationCenter()
 	decisionService := &CompositeService{
```

Comment:
Does the composite service need to know about the SDK key? Or, can the registry take care of that?